src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapMercury.java

```diff
@@ -1,36 +1,162 @@
 package com.wikia.webdriver.testcases.adstests;
 
+import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
 import com.wikia.webdriver.common.core.url.Page;
 import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
 import com.wikia.webdriver.common.templates.mobile.MobileTestTemplate;
 import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VideoFanTakeover;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 @Test(
         groups = "AdsVuapMercury"
 )
 public class TestAdsVuapMercury extends MobileTestTemplate {
 
-    private static final String TOP_VIDEO_URL = "https://pubads.g.doubleclick.net/gampad/ads?output=vast&env=vp&gdfp_req=1&unviewed_position_start=1&iu=%2F5441%2Fwka.life%2F_project43%2F%2Farticle%2Fmobile%2FMOBILE_TOP_LEADERBOARD";
-
+    private static final String URL_FIRSTQUARTILE = "ad_vast_point=firstquartile";
+    private static final String URL_MIDPOINT = "ad_vast_point=midpoint";
+    private static final int DELAY = 2;
+    private static final int VIDEO_START_TIME = 0;
 
     @NetworkTrafficDump(useMITM = true)
     @Test(
             dataProviderClass = AdsDataProvider.class,
-            dataProvider = "adsVUAPTopMercury",
+            dataProvider = "adsVuapMercury",
             groups = "AdsTopAdVideoClosesWhenFinishPlaysMercury"
     )
-    public void adsTopAdVideoClosesWhenFinishPlaysMercury(Page page, String slotName, String iframeId) {
+    public void adsTopAdVideoClosesWhenFinishPlaysMercury(Page page, String slotName, String iframeId, String videoUrl) {
```

Comment:
I'd change the name here as well. Similar idea to the one above: `adsTestTopAdVideoBehavior()` or something like that.